source/templates/committer-welcome.txt

```diff
@@ -0,0 +1,67 @@
+To: private@[PROJECT].a.o, NewCommitterID@apache.org
+Subject: Welcome, New Committer!
+
+[Committer Name] as you know, the ASF Infrastructure has set up your
+committer account with the username `[apacheID]`
+
+[If your project uses SVN]
+
+Please follow the instructions to set up your SSH,
+svn password, svn configuration, email forwarding, etc.
+https://www.apache.org/dev/#committers
+
+[If your project automatically adds committers as PMC members]
+Please subscribe to the [PROJECT] Project Management
+Committee mailing list private@[PROJECT].apache.org.
+[/If PMC]
+
+[PROJECT] has various resources at:
+  https://svn.apache.org/repos/asf/[PROJECT]
+
+[/If SVN]
+
+[If your project uses GitHub]
+You will now be able to merge approved PRs on GitHub for this project.
+
+You can manage your account settings at https://id.apache.org/
+[/If GitHub]
+
+As an ASF committer, you now also have commit access to specific
+sections of the ASF Foundation repository, as follows:
+
+The general "committers" at:
+https://svn.apache.org/repos/private/committers
```

Comment:
This one also probably should go at the end of the list. I even had no idea this repo exists and it seems that the activity there is only about adding new committers by the secretary and few other people, I am not sure if there is anything interesting in that repo for a new committer - it's >5 years for me as a committer and it's the first time I checked it out today.

While removing might seem excessive, maybe putting it at the end of the list makes sense.